tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -129,6 +146,11 @@ class RPageSink : public RPageStorage {
 protected:
    RNTupleWriteOptions fOptions;
 
+   /// Helper to zip pages and header/footer; comprises a 16MB unzip buffer.
```

Comment:
Is this size (16MB) configurable in any way?  If so, we probably should have a reference to the configuration mechanism (or hint to one) ... if not, we probably should (eventually) have one :)

Reply:
That's simply `kMAXZIPBUF`, I mention the constant now in the comment.